fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -3182,6 +3182,15 @@ public void setDumpNereidsMemo(boolean dumpNereidsMemo) {
         this.dumpNereidsMemo = dumpNereidsMemo;
     }
 
+    public void disableStrictConsistencyDmlOnce() throws DdlException {
+        if (!enableStrictConsistencyDml) {
+            return;
+        }
+        setIsSingleSetVar(true);
+        VariableMgr.setVar(this,
+                new SetVar(SessionVariable.ENABLE_STRICT_CONSISTENCY_DML, new StringLiteral("false")));
```

Comment:
maybe we should change the name of this session variable, since it is not used for strict sondistency insert anymore.

Reply:
ok, could change it at another PR,  maybe like `enable_tablet_sink_shuffle`